web/js/main.jsx

```diff
@@ -172,44 +174,27 @@ return(<div className="player">{player}</div>)
 
     switch(type){
       case 'host':
-        socket = new WebSocket(`ws://localhost:5000/ws?name=${this.state.playerName}`);
+        socket = new WebSocket(`ws://192.168.1.64:5000/ws?name=${this.state.playerName}`);
```

Comment:
your local IP is going to change, imo leave this as localhost in the code and just update locally as needed.

Reply:
or possible even define it as a global and reference as needed e.g.

```
const targetHost = 'localhost'

...
socket = new WebSocket(`ws://${targetHost}/ws?name=${this.state.playerName}`
```